docs/docs/constraints.md

```diff
@@ -28,108 +28,108 @@ Regex is allowed for LIKE and UNLIKE operators; to match ANY value, use the stri
 
 ## Operators
 
-### UNIQUE operator
+### CLUSTER operator
+**Value** (required): A rack ID, set of rack IDs, or a specific node.
 
-`UNIQUE` tells Marathon to enforce uniqueness of the attribute across all of an app's tasks. For example the following constraint ensures that there is only one app task running on each host:
+`CLUSTER` allows you to run all of your app's tasks on agent nodes that share a certain attribute. This is useful for example if you have apps with special hardware needs, or if you want to run them on the same rack for low latency.
 
 ``` bash
 $ curl -X POST -H "Content-type: application/json" localhost:8080/v2/apps -d '{
-    "id": "sleep-unique",
+    "id": "sleep-cluster",
     "cmd": "sleep 60",
     "instances": 3,
-    "constraints": [["hostname", "UNIQUE"]]
+    "constraints": [["rack_id", "CLUSTER", "rack-1"]]
   }'
 ```
 
-### CLUSTER operator
-
-`CLUSTER` allows you to run all of your app's tasks on agent nodes that share a certain attribute. This is useful for example if you have apps with special hardware needs, or if you want to run them on the same rack for low latency:
+You can also use this attribute to tie an application to a specific node by using the hostname property:
 
 ``` bash
 $ curl -X POST -H "Content-type: application/json" localhost:8080/v2/apps -d '{
     "id": "sleep-cluster",
     "cmd": "sleep 60",
     "instances": 3,
-    "constraints": [["rack_id", "CLUSTER", "rack-1"]]
+    "constraints": [["hostname", "CLUSTER", "a.specific.node.com"]]
   }'
 ```
 
-You can also use this attribute to tie an application to a specific node by using the hostname property:
+### LIKE operator
+**Value** (required): A rack ID or set of rack IDs.
```

Comment:
A regexp for the value of the attribute. This does not have to be that rack id (it is just an example)